_rules/css-orientation-lock-media-queries.md

```diff
@@ -0,0 +1,156 @@
+---
+name: Page has no orientation lock specified using CSS media queries
+
+rule_type: atomic
+
+description: |
+  This rule checks that page content is not locked to any specific display orientation using CSS media queries, and the content is operable in all display orientations.
+
+success_criterion:
+- 1.3.4 # (Orientation)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any `element` that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element does not have any `CSS` media queries applied, that locks the page content to a specific orientation.
```

Comment:
Mixing elements and page in the sentence makes it somewhat fuzzy. Given that the rule targets elements of the page, does it fail if I have an element of the page locked to a specific orientation, but the rest of the page is not locked?
My interpretation of the Success Criterion is that the locking restriction applies to the whole page and not to specific elements. 
The expectation is not clear because it reads "locks the page content". Does this equals to "locks the page" or to "locks some of the page's content"?

Reply:
Note:

Expand on the expectation - move the essential assumption into here and add a note with examples where orientation is essential